docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/svpc_liver-cancer-treatment.content.yml

```diff
@@ -173,7 +173,7 @@
   field_pdq_audience__ES:
     value: Patients
   field_pdq_summary_type__ES:
-    value: Overview
+    value: Treatment
```

Comment:
Why did this change?

Reply:
Bad YAML - Overview is not a valid option under summary type in drupal - updated yaml to match production